src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -209,12 +207,11 @@
 
 (defn community-content
   [{:keys [name description locked joined images
-           status tokens tags requested-to-join-at id]
-    :as   community}
+           status tokens tags id]
+    :as   community} pending?
    {:keys [on-categories-heights-changed
            on-first-channel-height-changed]}]
-  (let [pending?          (pos? requested-to-join-at)
-        thumbnail-image   (get-in images [:thumbnail])
+  (let [thumbnail-image   (get-in images [:thumbnail])
```

Comment:
`(:thumbnail images)`?

Reply:
That was a lazy refactor by me before. Will do it right this time 😁